### PR TITLE
Fix minor bugs

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java [hunk 1]
```diff
@@ -81,8 +81,8 @@ public void setconnectedCupMQTTConnector(
     @Path("controller/coffeelevel")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @Feature(code = "coffeelevel", name="Coffeelevel", description = "read coffeelevel from this device", type =
-            "monitor")
+    @Feature( code="coffeelevel", name="Coffee Level", type="monitor",
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java:84]: fix formatting issues on spacing.

File: modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java [hunk 2]
```diff
@@ -121,8 +121,8 @@ public SensorRecord readCoffeeLevel(@HeaderParam("owner") String owner,
     @Path("controller/temperature")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @Feature(code = "temperature", name="Temperature", description = "read temperature from this device", type =
-            "monitor")
+    @Feature( code="temperature", name="Temperature", type="monitor",
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java:124]: fix formatting issues on spacing.

File: modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js [hunk 3]
```diff
@@ -141,13 +141,13 @@ function downloadAgent() {
         values[this.name] = $(this).val();
     });
 
-    var connectedCupRegisterURL = "/connectedcup_mgt/connectedcup/cup/register?" +
-                                                        "name=" + payload.name + "&owner=" + payload.owner;
-
     var payload = {};
     payload.name = $inputs[0].value;
     payload.owner = $inputs[1].value;
```
Comment (DilanUA) [inline modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js:146]: Use double quoted object key access for reading JSON attributes as it's considered to be more reliable.
Ex: person.name as person["name"]

File: modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java (hunk 1 above)
Comment (charithag) [inline modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java:84]: Fixed in PR #179

File: modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java (hunk 2 above)
Comment (charithag) [inline modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/ConnectedCupControllerService.java:124]: Fixed in PR #179

File: modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js (hunk 3 above)
Comment (charithag) [inline modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js:146]: How bracket notation could be more reliable? It is all about the language semantics. Using dot notation is permitted in JS as long as attribute name is a valid identifier.
http://stackoverflow.com/a/4968448/1391687

**Pull request merged**